src/components/InstanceListView/index.tsx

```diff
@@ -4,79 +4,61 @@ import { Link } from "react-router-dom";
 import { IClusterServiceClass } from "../../shared/ClusterServiceClass";
 import { IServiceBroker, IServicePlan } from "../../shared/ServiceCatalog";
 import { IServiceInstance } from "../../shared/ServiceInstance";
-import { Card, CardContainer } from "../Card";
+import { InstanceTable } from "./InstanceTable";
 
 export interface InstanceListViewProps {
   brokers: IServiceBroker[];
   classes: IClusterServiceClass[];
   getCatalog: () => Promise<any>;
+  checkCatalogInstalled: () => Promise<boolean>;
   instances: IServiceInstance[];
   plans: IServicePlan[];
+  isInstalled: boolean;
 }
 
 export class InstanceListView extends React.PureComponent<InstanceListViewProps> {
   public async componentDidMount() {
+    this.props.checkCatalogInstalled();
     this.props.getCatalog();
   }
 
   public render() {
-    const { brokers, instances, classes } = this.props;
+    const { isInstalled, brokers, instances, classes } = this.props;
 
     return (
       <div className="InstanceList">
-        {brokers && (
+        <h3>Service Instances</h3>
+
+        {isInstalled ? (
           <div>
-            <div className="row">
-              <div className="col-8">
-                <h3>Service Instances</h3>
-                <p>Service instances from your brokers:</p>
-              </div>
-              <div className="col-2 margin-normal">
-                <Link to={`/services/classes`}>
-                  <button className="button button-primary">Provision New Service</button>
-                </Link>
+            {brokers.length > 0 ? (
+              <div>
+                <div className="row">
+                  <div className="col-8">
+                    <p>Service instances from your brokers:</p>
+                  </div>
+                  <div className="col-2 margin-normal">
```

Comment:
This is not aligned properly. Use `col-4 text-r` to place the button at the end of the row